common/test/acceptance/tests/lms/test_lms_course_home.py

```diff
@@ -167,73 +167,3 @@ def test_course_search_a11y(self):
             ]
         })
         course_search_results_page.a11y_audit.check_for_accessibility_errors()
-
-
-class CourseOutlineTest(UniqueCourseTest):
```

Comment:
Just deleting? Is new logic tested?

Reply:
Not sure if the particular thing this is testing (self-paced courses not getting dates/getting dates) is tested specifically, but the condition this test was looking for (self-paced courses don't get due dates) is no longer true, and the test wasn't appropriate for bok-choy anyway (it should have been written as an lms unit test)

Reply:
Oy, didn't notice it was a bok-choy file.